uc8151/registers.go

```diff
@@ -141,11 +141,6 @@ const (
 	HZ_100 = 0b00111010
 	HZ_200 = 0b00111001
 
-	NO_ROTATION  Rotation = 0
-	ROTATION_90  Rotation = 1 // 90 degrees clock-wise rotation
-	ROTATION_180 Rotation = 2
-	ROTATION_270 Rotation = 3
-
```

Comment:
That's a breaking change. If you're fine with it that's okay. In other packages I've basically aliased these constants like `NO_ROTATION = drivers.Rotation0`.

Reply:
That is probably a good idea. I will do so.